__tests__/search.test.js

```diff
@@ -1,7 +1,7 @@
 const { goTo, getProperty } = require('../lib/puppeteer-helpers.js')
 
 // Regex that can be used to match on fingerprinted search index files
-const isSearchIndex = /.*\/search-index-[0-9a-f]{32}.json$/
+const isSearchIndex = /.*\/search-index(-[0-9a-f]{32})?.json$/
```

Comment:
Makes the hash fingerprint optional `?` so matches `search-index.json` when running locally